src/components/general/Header.js

```diff
@@ -66,15 +66,20 @@ export default function Header() {
 
   return (
     <Box component="header" className={classes.root}>
-      <Container className={classes.container}>
-        <Link href="/">
-          <a>
-            <img src="/images/logo.png" alt="Climate Connect" className={classes.logo} />
-          </a>
-        </Link>
-
-        {isNarrowScreen ? <NarrowScreenLinks /> : <NormalScreenLinks />}
-      </Container>
+      {process.env.PRE_LAUNCH === "true" ? (
+        <Container className={classes.container}>
+          <img src="/images/logo.png" alt="Climate Connect" className={classes.logo} />
+        </Container>
+      ) : (
+        <Container className={classes.container}>
+          <Link href="/">
+            <a>
+              <img src="/images/logo.png" alt="Climate Connect" className={classes.logo} />
+            </a>
+          </Link>
+          {isNarrowScreen ? <NarrowScreenLinks /> : <NormalScreenLinks />}
```

Comment:
I think here it would be easier to understand if you just wrapped an if else ( ? : ) function around `{isNarrowScreen ? <NarrowScreenLinks /> : <NormalScreenLinks />}`.
Something like this: 
`process.env.PRE_LAUNCH === true ? </> : (isNarrowScreen ? <NarrowScreenLinks /> : <NormalScreenLinks />)`
I think this makes sense because the image should still be a link going to "/" in the PRE_LAUNCH version and therefore this is the only difference between PRE_LAUNCH=true and PRE_LAUNCH=false

Reply:
Makes sense, updated code as requested.